playbook/app/pb_kits/playbook/pb_advanced_table/_advanced_table.tsx

```diff
@@ -49,6 +58,9 @@ type AdvancedTableProps = {
   toggleExpansionIcon?: string | string[]
   onRowSelectionChange?: (arg: RowSelectionState) => void
   virtualizedRows?: boolean
+  fullscreenable?: boolean
+  onFullscreenChange?: (isFullscreen: boolean) => void
+  getFullscreenControls?: (controls: FullscreenControls) => void
```

Comment:
For callbacks of this sort, we tend to stick to the 'on' syntax for prop names eg: onIconChange, onSelect, etc. 

For this one, what do you think about `onFullScreenToggle`?